Modules/Models/Country.swift

```diff
@@ -0,0 +1,58 @@
+// Country.swift
+// Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+// Please refer to the AUTHORS file for more information.
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+import Foundation
+
+public struct Country: Equatable, Codable {
+    
+    public var countryId: String
+    public var countryHumanReadableName: String
+    
+    public init(countryId: String, countryHumanReadableName: String){
+        self.countryId = countryId
+        self.countryHumanReadableName = countryHumanReadableName
+    }
+}
+
+extension Country: Comparable {
+    
+    public static func ==(lhs: Self, rhs: Self) -> Bool {
+           return lhs.countryId == rhs.countryId
+       }
+    public static func < (lhs: Self, rhs: Self) -> Bool {
+        return lhs.countryHumanReadableName < rhs.countryHumanReadableName
+    }
+   
```

Comment:
<!--
  0 failure: 
  2 warning:  swiftlint (0.39.2..., swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Country.swift;
  Line: 37;
-->

- :warning: swiftlint (0.39.2): Don't include vertical whitespace (empty line) before closing braces. (vertical_whitespace_closing_braces)
- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)